### PR TITLE
Revert "Deprecate `az sf` commands (#4323)"

### PR DESCRIPTION
**Pull request by QingChenmsft**
This reverts commit bfd7d99f8097410b94b006ed074773610cc97251.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-sf/HISTORY.rst [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 Release History
 ===============
+<<<<<<< HEAD
+
+1.0.9 (2017-09-14)
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-sf/HISTORY.rst]: Heading should be `unreleased`.
Also, `<<<<<<< HEAD` should not be there.

Comment (QingChenmsft) [reply src/command_modules/azure-cli-sf/HISTORY.rst]: fixed.

File: src/command_modules/azure-cli-sf/HISTORY.rst [hunk 2]
```diff
@@ -1,10 +1,12 @@
 .. :changelog:
 
-Release History
+Unreleased History
 ===============
 
+1.0.9 (2017-09-14)
+* Revert 'Deprecating all commands in favor of Service Fabric CLI (sfctl)'
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-sf/HISTORY.rst]: That's not what I meant.
More like this...

```
Release History
===============

unreleased
++++++++++
* Revert 'Deprecating all commands in favor of Service Fabric CLI (sfctl)'
```

see https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log

File: src/azure-cli-core/azure/cli/core/sdk/util.py [hunk 3]
```diff
@@ -57,7 +57,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         pass
 
     def command(self, name, method_name, transform=None, table_transformer=None, confirmation=None,
-                exception_handler=None, deprecate_info=None, no_wait_param=None):
+                exception_handler=None, no_wait_param=None):
```
Comment (samedder) [inline src/azure-cli-core/azure/cli/core/sdk/util.py]: It doesn't make sense to revert these changes. It's optional parameters. Rather than just straight attempting to revert the commit, take some time to remove the deprecation string from the sf commands.

Comment (QingChenmsft) [reply src/azure-cli-core/azure/cli/core/sdk/util.py]: no, this is introduced by this change, prefer to revert at once.

**Comment by samedder** [comment]
@QingChenmsft if these commands are going to be removed they need to be removed entirely, rather than just reverting the deprecation message.
